.pullapprove.yml

```diff
@@ -63,8 +63,8 @@ groups:
 
   ux-core:
     conditions:
-    - "'**/*.hbs' in files or 'static/styles/**/*.scss' in files or '.pullapprove.yml' in files"
-    # only review if "**/*.hbs", "static/styles/**/*.scss" files or ".pullapprove.yml" have been touched
+    - "'static/images/*/.' in files, theme/* or 'static/fonts/*/.' in files or 'static/styles//.scss' in files"
+    # only review if "'static/images/*/.', theme/*, 'static/fonts/*/.', 'static/styles//.scss'" files have been touched
```

Comment:
I'm not sure if this will work.
Please re-check:
```
- "'static/images/**/*.*' in files or 'static/fonts/**/*.*' in files or 'static/styles/**/*.scss' in files or 'theme/**/*.scss' in files or 'theme/*/static/**/*.*' in files"
# only review if "static/images/**/*.*", "static/fonts/**/*.*", "static/styles/**/*.scss", "theme/**/*.scss", or "theme/*/static/**/*.*" have been touched"
```

I've tried those patterns with
- `find static/images/**/*.*`
- `find static/fonts/**/*.*`
- `find static/styles/**/*.scss`
- `find theme/**/*.scss'`
- `find theme/*/static/**/*.*`

within my local bash.